src/argparse.ts

```diff
@@ -2271,6 +2271,33 @@ export const CLI_ARGS = {
       '    ]\n' +
       '\n',
       group: 'Peer Services'
+    },
+    get_did_configuration: {
+      type: 'array',
+      items: [
+        {
+          name: 'blockstack_id',
+          type: 'string',
+          realtype: 'blockstack_id',
+          pattern: NAME_PATTERN + '|'+ SUBDOMAIN_PATTERN
+        },
+        {
+          name: 'domain',
+          type: 'string',
+          realtype: 'domain',
+          pattern: NAME_PATTERN + '|'+ SUBDOMAIN_PATTERN
+        },
+        {
+          name: 'owner_key',
+          type: 'string',
+          realtype: 'private_key',
+          pattern: `${PRIVATE_KEY_PATTERN_ANY}`
```

Comment:
Is this supposed to take a single private key, or can it take a collection of private keys?  This regex matches the latter, but the code only handles the case where it's a single private key.

Reply:
I am using PRIVATE_KEY_PATTERN now, becaues get_owner_keys returns the key as string of length 66.